scripts/check.js

```diff
@@ -173,11 +173,11 @@ export default () => new Promise(async (resolve, reject) => {
     }
 
     log.info('Offline STT')
-    if (!fs.existsSync(deepSpeechPath)) {
+    if (!fs.existsSync(coquiPath)) {
       report.can_offline_stt.v = false
-      log.warning(`Cannot find ${deepSpeechPath}. You can setup the offline STT by running: "npm run setup:offline-stt"`)
+      log.warning(`Cannot find ${coquiPath}. You can setup the offline STT by running: "npm run setup:offline-stt"`)
     } else {
-      log.success(`Found DeepSpeech language model at ${deepSpeechPath}`)
+      log.success(`Found DeepSpeech language model at ${coquiPath}`)
```

Comment:
nit: `s/DeepSpeech language model/Coqui acoustic model/`

Coqui (like DeepSpeech) has both an acoustic model and a language model. The acoustic model is `model.tflite` and the language model ends in `*.scorer`. Here you're checking for `coquiPath`, which has been set to the acoustic model